### PR TITLE
Change parser for Various VEIKK tablets

### PR DESCRIPTION
**Pull request by jamesbt365**
If i have done this wrong, blame void for not giving me more instructions. These changes are untested.

### CODE REVIEW

**Comment by jamesbt365** [comment]
for reasoning behind adding these inits: https://canary.discord.com/channels/615607687467761684/615607687467761690/986623762789191730

**Comment by jamesbt365** [comment]
Only Veikk A30 v1 and A50 verification is needed now. Veikk vk430 should cover vk640.

File: OpenTabletDriver.Configurations/Parsers/Veikk/VeikkAuxReport.cs [hunk 1]
```diff
@@ -9,10 +9,15 @@ public VeikkAuxReport(byte[] report)
...
-                report[4].IsBitSet(2) && report[3].IsBitSet(0), // Third
-                report[4].IsBitSet(3) && report[3].IsBitSet(0), // Fourth
+                report[4].IsBitSet(0) && report[3].IsBitSet(0),
+                report[4].IsBitSet(1) && report[3].IsBitSet(0),
+                report[4].IsBitSet(2) && report[3].IsBitSet(0),
+                report[4].IsBitSet(3) && report[3].IsBitSet(0),
+                report[4].IsBitSet(4) && report[3].IsBitSet(0),
+                report[4].IsBitSet(5) && report[3].IsBitSet(0),
+                report[4].IsBitSet(6) && report[3].IsBitSet(0),
+                report[4].IsBitSet(7) && report[3].IsBitSet(0),
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Parsers/Veikk/VeikkAuxReport.cs:19]: Do these changes on a separate PR

**Comment by X9VoiD** [comment]
Marking as draft until it's actually tested, the new parser has an entirely different format for Y position.

File: OpenTabletDriver.Configurations/Parsers/Veikk/VeikkAuxReport.cs (hunk 1 above)
Comment (jamesbt365) [reply OpenTabletDriver.Configurations/Parsers/Veikk/VeikkAuxReport.cs]: resolved in #2334

**Comment by jamesbt365** [comment]
VK640 verification: https://discord.com/channels/615607687467761684/787072060563128321/1018179710368366613
Safe to say the vk430 also will work then

The a30 and a50 no clue on yet though with different format for positioning its unlikely they'll work so probably best to rip them from this pr before i rebase.

**Comment by jamesbt365** [comment]
Rebased.

Removing veikk A30 and A50 from this pull request due to no verification that it works.
